addon/components/property-group.hbs

```diff
@@ -1,13 +1,19 @@
 {{#if this.children}}
   <div ...attributes>
-    {{#if @group.name}}
+    {{#if this.showTitleBlock}}
```

Comment:
This change also allows us to show help text even if we don't want to show a title. This should remove the need to hack around it by setting the title to an empty string (`sh:name ""`)